lib/container_orchestrator/object_definition.rb

```diff
@@ -179,6 +189,25 @@ def default_environment
       ] + database_environment + memcached_environment + messaging_environment
     end
 
+    SECRET_KEY_FILE = "/run/secrets/manageiq/application/encryption_key".freeze
+    def opentofu_runner_secret_key
+      @opentofu_runner_secret_key ||= File.exist?(SECRET_KEY_FILE) ? File.read(SECRET_KEY_FILE) : "opentofu_runner_key"
+    end
+
+    TERRAFORM_RUNNER_TOKEN_FILE = Rails.root.join("TERRAFORM_RUNNER_TOKEN").freeze
+    def opentofu_runner_token
+      secret_key = opentofu_runner_secret_key
+      payload = {
+        'Username' => 'opentofu-runner'
+      }
+      jwtToken = JWT.encode(payload, secret_key, 'HS256')
+      File.open(TERRAFORM_RUNNER_TOKEN_FILE, "wb") do |file|
+        file.sync = true
+        file.write(jwtToken)
+      end  
+      @opentofu_runner_token ||=  TERRAFORM_RUNNER_TOKEN_FILE
```

Comment:
Do we need an instance var for a frozen constant?  Or did you intend to set this to the value in `jwtToken` ?